AsyncDisplayKit/ASTextNode.h

```diff
@@ -71,6 +91,12 @@ typedef NS_ENUM(NSUInteger, ASTextNodeHighlightStyle) {
 @property (nonatomic, assign) NSUInteger maximumNumberOfLines;
 
 /**
+ @abstract The vertical alignment of text within the receiver.
```

Comment:
Use the documentation flag "default" to indicate the default value.  In the discussion, say something like:

"This property should not be used as the primary mechanism to vertically position text, which should be achieved with layout specs or the -layout method.  However, in cases where it is necessary to alter the position at which text draws within the text node's backing store, this can be used.  Note: this scenario only occurs if the text node's bounds / frame are set to a size greater than specified by calculatedLayout (or return value from -measureWithSizeRange:).  Over-sizing the text node results in extra memory usage and compositing, so consider alternative solutions before relying on .verticalTextAlignment"